### PR TITLE
webgpu: Update @webgpu/types to 0.0.38

### PR DESCRIPTION
**Pull request by qjia7**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/4723)

### CODE REVIEW

**Comment by qjia7** [comment]
@kainino0x @lina128 Please take a look, thanks.

File: tfjs-backend-webgpu/src/kernels/FromPixelsImageBitmap.ts [hunk 1]
```diff
@@ -60,7 +60,11 @@ export function fromPixelsImageBitmap(args: {
         texture: backend.fromPixelProgram.makeInputTexture(
             backend.device, imageBitmap.width, imageBitmap.height)
       },
-      {width: imageBitmap.width, height: imageBitmap.height, depth: 1});
+      {
+        width: imageBitmap.width,
+        height: imageBitmap.height,
+        depthOrArrayLayers: 1
```
Comment (kainino0x) [inline tfjs-backend-webgpu/src/kernels/FromPixelsImageBitmap.ts:66]: nit: It should be possible to omit `depthOrArrayLayers` as it's optional. You can also write it as an array if you want, `[imageBitmap.width, imageBitmap.height]`.

File: tfjs-backend-webgpu/src/kernels/FromPixels_utils/from_pixels_webgpu.ts [hunk 2]
```diff
@@ -136,7 +135,7 @@ export class FromPixelsProgram implements WebGPUProgram {
         size: {
           width: pixelWidth,
           height: pixelHeight,
-          depth: 1,
+          depthOrArrayLayers: 1,
```
Comment (kainino0x) [inline tfjs-backend-webgpu/src/kernels/FromPixels_utils/from_pixels_webgpu.ts:138]: same

**Review by kainino0x — APPROVED** [review]
LGTM

**Review by lina128 — APPROVED** [review]
LGTM

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/4723#-:-MUA8h2XESZbPXYJ2eql:bb6ywmx)* status: :shipit: complete! 2 of 1 approvals obtained (waiting on @qjia7)

File: tfjs-backend-webgpu/src/kernels/FromPixelsImageBitmap.ts (hunk 1 above)
Comment (qjia7) [reply tfjs-backend-webgpu/src/kernels/FromPixelsImageBitmap.ts]: Done.

File: tfjs-backend-webgpu/src/kernels/FromPixels_utils/from_pixels_webgpu.ts (hunk 2 above)
Comment (qjia7) [reply tfjs-backend-webgpu/src/kernels/FromPixels_utils/from_pixels_webgpu.ts]: Done.

**Pull request merged**